subprojects/wrapper/build.gradle.kts

```diff
@@ -33,17 +37,36 @@ dependencies {
 }
 
 val executableJar by tasks.registering(Jar::class) {
-    archiveFileName = "gradle-wrapper.jar"
+    archiveFileName = "gradle-wrapper-executable.jar"
     manifest {
         attributes.remove(Attributes.Name.IMPLEMENTATION_VERSION.toString())
         attributes(Attributes.Name.IMPLEMENTATION_TITLE.toString() to "Gradle Wrapper")
     }
     from(sourceSets.main.get().output)
-    from(configurations.runtimeClasspath.get().incoming.artifactView {
-        attributes.attribute(LibraryElements.LIBRARY_ELEMENTS_ATTRIBUTE, objects.named(LibraryElements.CLASSES))
-    }.files)
+    exclude("gradle-*-classpath.properties")
```

Comment:
🤔 Are these excludes required here, since we add them also to gr8?

Reply:
Gr8 shades resources from dependencies, here we are filtering out resources from the current project.